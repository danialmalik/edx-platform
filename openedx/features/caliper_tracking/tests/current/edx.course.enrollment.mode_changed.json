{
   "username":"honor",
   "event_source":"server",
   "name":"edx.course.enrollment.mode_changed",
   "accept_language":"en-US,en;q=0.9",
   "time":"2018-11-16T07:52:15.893562+00:00",
   "agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
   "page":null,
   "host":"8c3b43aa7b1b",
   "session":"e826193d6f48613a115e4387d12e4d5d",
   "referer":"http://localhost:18000/courses/course-v1:PUCIT+CS520+2018_T4/about",
   "context":{
      "user_id":6,
      "org_id":"PUCIT",
      "course_id":"course-v1:PUCIT+CS520+2018_T4",
      "path":"/change_enrollment"
   },
   "ip":"172.18.0.1",
   "event":{
      "course_id":"course-v1:PUCIT+CS520+2018_T4",
      "user_id":6,
      "mode":"honor"
   },
   "event_type":"edx.course.enrollment.mode_changed"
}
