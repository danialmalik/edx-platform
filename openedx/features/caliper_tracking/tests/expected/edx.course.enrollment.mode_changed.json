{
   "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
   "id":"urn:uuid:921bd866-93ac-4ed1-a147-cdd1fe91e87f",
   "type":"Event",
   "actor":{
      "type":"Person",
      "name":"honor",
      "id":"edx.devstack.lms:18000/u/honor"
   },
   "action":"Modified",
   "object":{
      "id": "http://localhost:18000/courses/course-v1:PUCIT+CS520+2018_T4/about",
      "type": "Membership"
   },
   "eventTime":"2018-11-16T07:52:15.893Z",

   "referrer":"http://localhost:18000/courses/course-v1:PUCIT+CS520+2018_T4/about",

   "extensions":{
      "extra_fields":{
         "event_source":"server",
         "accept_language":"en-US,en;q=0.9",
         "session":"e826193d6f48613a115e4387d12e4d5d",
         "ip":"172.18.0.1",
         "user_id":6,
         "org_id":"PUCIT",
         "event_type":"edx.course.enrollment.mode_changed",
         "agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
         "course_id":"course-v1:PUCIT+CS520+2018_T4",
         "host":"8c3b43aa7b1b",
         "path":"/change_enrollment",
         "mode":"honor",
         "page":null
      }
   }
}
